<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChallengeBoard</class>
 <widget class="QWidget" name="ChallengeBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="input">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tips">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;请输入单词后按回车键或确认来完成猜题&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_confirm">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>380</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_switch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>去出题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_logout">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>600</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_hi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>那谁，你好！</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_stage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>你已经闯到第x关</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_level">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>你的闯关者等级为 x 级</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_exp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>目前经验进度为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="tips_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>600</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>本次挑战累计时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_time">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>600</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>0 secs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_search">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_start">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>30</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_word">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>571</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Demi ITC</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Eras Demi ITC&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;hilarious&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_worddis">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Eras Demi ITC</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Eras Demi ITC&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="B_skip">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>380</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>跳过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tips_skip">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>点我跳过</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_save">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
